src/injectscripts/messages/create.js

```diff
@@ -125,72 +131,160 @@ $(document).ready(function() {
                 currentCollections.push(CollectionParent);
                 localStorage.setItem("lighthouseContactCollections", JSON.stringify(currentCollections));
                 LoadAllCollections();
-            }            
+            }
         }
     })
 
 
-//Load all the Collections on page load
+    //Load all the Collections on page load
 
-LoadAllCollections();
+    LoadAllCollections();
 
-DoTour()
+    DoTour()
 
 
 
 });
 
+
+function LoadTeams() {
+    $('#lighthouseteams').empty() //empty to prevent dupes
+
+    var $spinner = (<i style="width:100%; margin-top:4px; margin-left:auto; margin-right:auto" class="fa fa-refresh fa-spin fa-2x fa-fw"></i>)
```

Comment:
don't name a var starting with a `$` unless it is already a jquery object. Adding a `$` in front of the `(` would fix this. Then remove the `$(...)` around `$spinner` on the next line.